ql/src/java/org/apache/hadoop/hive/ql/udf/UDFFromUnixTime.java

```diff
@@ -31,12 +30,14 @@
 /**
  * UDFFromUnixTime.
  *
+ * @deprecated Replaced by {@link org.apache.hadoop.hive.ql.udf.generic.GenericUDFFromUnixTime}.
```

Comment:
Yes, that's a good point. I will remove it from the patch for master and I will keep it for the backport to 3.x branch in case users are referencing the UDF directly in their queries.